optaplanner-examples/src/main/java/org/optaplanner/examples/common/domain/AbstractPersistableJackson.java

```diff
@@ -0,0 +1,30 @@
+package org.optaplanner.examples.common.domain;
+
+import org.optaplanner.core.api.domain.lookup.PlanningId;
+
+public abstract class AbstractPersistableJackson {
```

Comment:
Why do we need this class? I don't see the `AbstractPersistable` using anything from XStream.

Reply:
Until all examples are converted, I'll keep this. I'm making some changes to this class and every example needs to be adapted.